File: Python/practice.py
problemName = 'practiceproblem'

#file = open('..\\StudentData\\' + problemName + '.dat', 'r')
file = open('H:\\JudgesData\\' + problemName + '.dat', 'r')

numLines = int(file.readline())

for i in range(numLines):
    print 'Let\'s play', file.readline().strip() + "!"

File: Java/src/Practice.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;

public class Practice {
    
    public static void main(String[] args) throws IOException {
        String problemName = "practiceproblem";
//        BufferedReader reader = new BufferedReader(new FileReader("..\\StudentData\\" + problemName + ".dat"));
        BufferedReader reader = new BufferedReader(new FileReader("H:\\JudgesData\\" + problemName + ".dat"));
        
        int numLines = Integer.valueOf(reader.readLine());
        
        for (int i = 0; i < numLines; i++) {
            System.out.println("Let's play " + reader.readLine() + "!");
        }
    }
}
